nextcord/application_command.py

```diff
@@ -917,11 +962,10 @@ async def invoke_error(self, interaction: Interaction, error: Exception) -> None
         """|coro|
         Invokes the error handler if available.
         """
-        if self.has_error_handler() and self.error_callback is not None:
-            if self.parent_cog:
-                await self.error_callback(self.parent_cog, interaction, error)
-            else:
-                await self.error_callback(interaction, error)
+        if (
+            self.has_error_handler() and self.error_callback is not None
+        ):  # TODO Why is this here? self.has_error_handler does the same as the second check
+            await self.error_callback(interaction, error)
```

Comment:
How does this handle passing `self` to callbacks if required? Just seems to be something from the removed code, so I'm just curious if that is still applicable here or not

Reply:
It automatically does. The code I removed didn't work.
It passed self automatically, than another self via the object and than the rest args.
Which gave me the error of having 4 args when 3 were exspected.

The code I replaced it with works.